merge_data.py

```diff
@@ -126,13 +126,13 @@ def merge_data():
     data = data.reset_index(drop=True)
 
     ### Saves copy of data without cleaning - for analysis purposes
-    data.to_csv("data/merged_output_untidy.csv", index=False)
+    data.to_json("data/merged_output_untidy.json", orient="index", date_format="iso")
```

Comment:
Would we be better using `orient="records"` for these rather than `index`?

Adding an index key for each record seems a bit unnecessary.

Reply:
I just like seeing the numbers... 👀

Have changed to `orient='records'` which removes index.